node/blockchain/admin.py

```diff
@@ -12,4 +12,4 @@ class BlockAdmin(admin.ModelAdmin):
 
 @admin.register(AccountState)
 class AccountStateAdmin(admin.ModelAdmin):
-    pass
+    list_display = list(f.name for f in AccountState._meta._get_fields())
```

Comment:
let's hardcode field names, also I do not think we need all of them, `_id` should be enough